pkgs/development/libraries/science/biology/elastix/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   pname    = "elastix";
-  pversion = "4.9.0";
+  pversion = "5.0.0";
   name  = "${pname}-${pversion}";
 
   src = fetchurl {
     url    = "https://github.com/SuperElastix/${pname}/archive/${pversion}.tar.gz";
```

Comment:
Don't use fetchurl on GH _archives_ as they get regenerated from time to time and the hash may change.
You should use `fetchFromGitHub` or  `fetchzip` instead.

Reply:
Good catch, thanks.